redaxo/src/core/lib/util/string.php

```diff
@@ -186,7 +186,7 @@ public static function buildQuery(array $params, $argSeparator = '&')
                 if (is_array($value)) {
                     $func($value, $key);
                 } else {
-                    $query[] = $key . '=' . str_replace('%2F', '/', urlencode($value));
+                    $query[] = $key . '=' . str_replace('%2F', '/', urlencode($value ?? ''));
```

Comment:
ich denke wir brauchen den gleichen fix noch 2x in zeile 185, da wir auch dort die typen nicht kennen

Reply:
`$query[] = $key . '=' . str_replace('%2F', '/', urlencode((string) $value));`
ginge auch. null gecastet ist ja '', oder ? Dann wäre psalm zufrieden.
Aber vielleicht ändert sich das in PHP 13 :-)

Reply:
String cast macht sinn denke ich